xbmc/pvr/addons/PVRClients.h

```diff
@@ -28,6 +28,8 @@
 
 #include "addons/PVRClient.h"
 
+#include "epg/EpgInfoTag.h"
```

Comment:
If I do a forward declaration, I get this error:
`PVRClients.cpp: In member function ‘bool PVR::CPVRClients::IsRecordable(const EPG::CEpgInfoTag&) const’:
PVRClients.cpp:1462:20: error: invalid use of incomplete type ‘const class EPG::CEpgInfoTag’
   if (GetClient(tag.ChannelTag()->ClientID(), client))
                    ^
In file included from PVRClients.h:26:0,
                 from PVRClients.cpp:21:
PVRChannel.h:39:9: note: forward declaration of ‘class EPG::CEpgInfoTag’
   class CEpgInfoTag;
         ^
`
Am I doing something wrong here?